src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1806,6 +1806,8 @@ internal enum ErrorCode
         ERR_InvalidFuncPointerReturnTypeModifier = 8797,
         ERR_DupReturnTypeMod = 8798,
         ERR_AddressOfMethodGroupInExpressionTree = 8799,
+        ERR_CannotConvertAddressOfToDelegate = 8800,
+        ERR_AddressOfToNonFunctionPointer = 8801,
```

Comment:
The error codes appear to be used already. (Most are used in master and 8801 is used in [https://github.com/dotnet/roslyn/pull/44636](https://github.com/dotnet/roslyn/pull/44636).)

Reply:
I'm already going to have to resolve conflicts here, one or two more isn't particularly more difficult.

---
In reply to: [432171912](https://github.com/dotnet/roslyn/pull/44613#discussion_r432171912) [](ancestors = 432171912)